### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by milesmcc**
#### Before submitting the PR
- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Make sure that the change you propose is the smallest possible.
- [x] Screenshots are included if the PR changes the UI.
- [x] If you change the UI, make sure the user experience is consistent with the current interface.

#### Code-related tasks **N/A**
- [ ] Tests added for this feature/bug.
- [ ] Impact on the seeders.
- [ ] Impact on the API.

#### If the code changes the SQL schema **N/A**
- [ ] Impact on account export.
- [ ] Impact on importing data with `vCard` and `.csv` files.
- [ ] Impact on account reset and deletion.

#### Other tasks
- [ ] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG) entry added, if necessary, under `UNRELEASED`. **N/A**
- [ ] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary. **N/A, no need; this is just a typo fix**
- [ ] If it's relevant and worth mentioning, create a changelog entry for this change. The changelog entry will appear inside the UI for all users to see. To know if your change is worth the creation of a changelog entry, [read the documentation](https://github.com/monicahq/monica/blob/master/docs/administrators/tips.md#when-is-it-relevant-to-create-a-changelog-entry). **N/A**
- [x] Indicate `[wip]` in the title of the PR it is is not final yet. Remove `[wip]` when ready. Otherwise the PR will be considered complete and rejected if it's not working.
- [x] Don't forget to [ask for a free account](mailto:regis@monicahq.com) on https://monicahq.com as anyone who contributes can request a free account. **N/A; I already have an account :)**

This pull request fixes a typo in the README: mismatched parenthesis. Prior to this PR, there were two open parenthesis, but only one closed parenthesis. This change makes a small change to the text to fix this error.

Funnily enough, it's in the section about typo fixes not counting toward free 'premium' accounts. Well, the irony is not lost on me! But I am not requesting such an account—I just want to fix the issue. :smile:

### CODE REVIEW

**Comment by milesmcc** [comment]
I don't think `tests-browser-7.2` failing was a result of this PR :slightly_smiling_face:

File: readme.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ If you want to have it for free with all the features, run the project yourself
     * Advanced features
     * Email reminders
     * Importers
-    * People who contribute to the GitHub repository (with a pull request that adds value, that gets merged (not a typo fix, for instance) will also have access to the Paid version for free.
+    * People who contribute to the GitHub repository (with a pull request that adds value, that gets merged--not a typo fix, for instance) will also have access to the Paid version for free.
```
Comment (asbiin) [inline readme.md]: Thanks! It will be even better with an en dash '–' caracter.

Comment (milesmcc) [reply readme.md]: @asbiin: thanks for the feedback. I think that an 'em dash' would be [more appropriate here](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0002.html), as en dashes are typically used to denote a range, i.e. January–September, as opposed to em dashes, which are typically used for sentence breaks.

**In any case, the first commit in this PR attempted to add an em dash (unicode 2014), but GitHub did not render the character. (See diff b54ec4d.) My takeaway was that GitHub doesn't want non-ASCII characters in the file, which is why I used two dashes.**

I changed the file back to an em dash in ecbf4cc; let me know if you can see the updated character. If not, we can look at a different solution. _This may be a bug in GitHub's online markdown editor._

**Comment by milesmcc** [comment]
@asbiin: thanks for the feedback. I think that an 'em dash' would be [more appropriate here](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0002.html), as en dashes are typically used to denote a range, i.e. January–September, as opposed to em dashes, which are typically used for sentence breaks.

**In any case, the first commit in this PR attempted to add an em dash (unicode 2014), but GitHub did not render the character. (See diff b54ec4d.) My takeaway was that GitHub doesn't want non-ASCII characters in the file, which is why I used two dashes.**

I changed the file back to an em dash in ecbf4cc; let me know if you can see the updated character. If not, we can look at a different solution. _This may be a bug in GitHub's online markdown editor._

**Comment by asbiin** [comment]
@milesmcc I can't see the em dash character here, not because of a GitHub bug, but because there is not! Did you used the GitHub editor inline ?
Just try to copy/paste the character from here '—' ?

**Comment by milesmcc** [comment]
Anything now? :slightly_smiling_face:

**Comment by asbiin** [comment]
@milesmcc great! Thanks I'll merge it.

**Pull request merged**